models/handpose_estimation_mediapipe/demo.py

```diff
@@ -0,0 +1,155 @@
+parser.add_argument('--vis', '-v', type=str2bool, default=True, help='Set true to open a window for result visualization. This flag is invalid when using camera.')
+args = parser.parse_args()
+
+
+def visualize(image, conf, hand_box, hand_landmarks, fps=None):
+    output = image.copy()
+
+    if fps is not None:
+        cv.putText(output, 'FPS: {:.2f}'.format(fps), (0, 15), cv.FONT_HERSHEY_SIMPLEX, 0.5, (0, 0, 255))
+
+    landmarks = hand_landmarks.reshape(21, 2).astype(np.int32)
+
+    # Draw line between each key points
+    cv.line(output, landmarks[0], landmarks[1], (255, 255, 255), 2)
+    cv.line(output, landmarks[1], landmarks[2], (255, 255, 255), 2)
+    cv.line(output, landmarks[2], landmarks[3], (255, 255, 255), 2)
+    cv.line(output, landmarks[3], landmarks[4], (255, 255, 255), 2)
+
+    cv.line(output, landmarks[0], landmarks[5], (255, 255, 255), 2)
+    cv.line(output, landmarks[5], landmarks[6], (255, 255, 255), 2)
+    cv.line(output, landmarks[6], landmarks[7], (255, 255, 255), 2)
+    cv.line(output, landmarks[7], landmarks[8], (255, 255, 255), 2)
+
+    cv.line(output, landmarks[0], landmarks[9], (255, 255, 255), 2)
+    cv.line(output, landmarks[9], landmarks[10], (255, 255, 255), 2)
+    cv.line(output, landmarks[10], landmarks[11], (255, 255, 255), 2)
+    cv.line(output, landmarks[11], landmarks[12], (255, 255, 255), 2)
+
+    cv.line(output, landmarks[0], landmarks[13], (255, 255, 255), 2)
+    cv.line(output, landmarks[13], landmarks[14], (255, 255, 255), 2)
+    cv.line(output, landmarks[14], landmarks[15], (255, 255, 255), 2)
+    cv.line(output, landmarks[15], landmarks[16], (255, 255, 255), 2)
+
+    cv.line(output, landmarks[0], landmarks[17], (255, 255, 255), 2)
+    cv.line(output, landmarks[17], landmarks[18], (255, 255, 255), 2)
+    cv.line(output, landmarks[18], landmarks[19], (255, 255, 255), 2)
+    cv.line(output, landmarks[19], landmarks[20], (255, 255, 255), 2)
+
+    for p in landmarks:
+        cv.circle(output, p, 2, (0, 0, 255), 2)
+
+    return output
+
+
+if __name__ == '__main__':
+    # palm detector
+    palm_detector = MPPalmDet(modelPath='../palm_detection_mediapipe/palm_detection_mediapipe_2022may.onnx',
+                              nmsThreshold=0.3,
+                              scoreThreshold=0.8,
+                              backendId=args.backend,
+                              targetId=args.target)
+    # handpose detector
+    handpose_detector = MPHandPose(modelPath=args.model,
+                                   confThreshold=args.conf_threshold,
+                                   backendId=args.backend,
+                                   targetId=args.target)
+
+    # If input is an image
+    if args.input is not None:
+        image = cv.imread(args.input)
+
+        # Palm detector inference
+        score, palm_box, palm_landmarks = palm_detector.infer(image)
```

Comment:
The returns need to be changed as well once https://github.com/opencv/opencv_zoo/pull/85 is merged.